packages/integration-tests/tests/connectors.test.ts

```diff
@@ -194,8 +195,23 @@ test('connector flow', async () => {
   expect(enabledEmailConnector.length).toEqual(1);
   expect(enabledEmailConnector[0]?.id).toEqual(sendgridEmailConnectorId);
 
+  /*
+   * Validate the connector config before updating it
+   *
+   * It should update the connector config successfully after the config passes the validation.
```

Comment:
Also, I think "validate" implies fail if not valid... please reorg your comments